src/fatx.h

```diff
@@ -20,7 +20,11 @@
 #ifndef FATX_H
 #define FATX_H
 
-#define _XOPEN_SOURCE
+#if __STDC_VERSION__ >= 199901L
```

Comment:
Why the branch? Just using 500 seems to work.

Reply:
I found this bit on [Stackoverflow](https://stackoverflow.com/questions/3875197/linux-gcc-with-std-c99-complains-about-not-knowing-struct-timespec), but maybe it's better to keep it more deterministic.

On OS X a value of 600 is required [for strdup](https://gitlab.labs.nic.cz/kslany/libisds/commit/9343ebfc1447952a6c0c601f99147ef969cdddeb) to be available (which is not used in the code), yet I don't have such a system to see if there are any other quirks.

If you prefer only one value, I'd suggest 600 just to be sure.